pkg/feature/templates/serverless/serving-istio-gateways/local-gateway-svc.tmpl

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    experimental.istio.io/disable-gateway-port-translation: "true"
+  name: {{ .Serving.LocalGatewayServiceName }} # knative-local-gateway
+  namespace: {{ .Mesh.Namespace }}
+spec:
+  ports:
+    - name: http2
```

Comment:
@israel-hdez please check out our updated installation docs, these contain an mTLS gateway: https://openshift-knative.github.io/docs/docs/latest/serverless/service-mesh/common-service-mesh-setup.html

Reply:
Also here as a diff for the original KServe setup: https://github.com/ReToCode/knative-kserve/commit/b7a8e720e8f37cc55dec5c5ea36eb53fb0fc8fb2

Reply:
The TLS gateway fixed [the other issue](https://github.com/opendatahub-io/opendatahub-operator/pull/691#discussion_r1379668050) :+1: